libclc/CMakeLists.txt

```diff
@@ -429,7 +411,9 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
     set( LIBCLC_ARCH_OBJFILE_DIR "${LIBCLC_OBJFILE_DIR}/${arch_suffix}" )
     file( MAKE_DIRECTORY ${LIBCLC_ARCH_OBJFILE_DIR} )
 
-    list( APPEND build_flags -cl-std=${opencl_lang_std} )
+    # Build for OpenCL 3.0 and enable all extensions and features independently
+    # of the target or device.
+    list( APPEND build_flags -cl-std=CL3.0 -Xclang -cl-ext=+all )
```

Comment:
The extensions should be enabled locally where needed

Reply:
That makes sense. Then targets could choose not to build builtins for which they don't support the appropriate extension.

Reply:
done, thanks